src/minimap.cpp

```diff
@@ -115,7 +115,7 @@ void MinimapUpdateThread::Stop()
 
 void MinimapUpdateThread::enqueue_Block(v3s16 pos, MinimapMapblock *data)
 {
-	if (m_queue.addBlock(pos, data))
```

Comment:
I've made the queue private in 420125debd3d010fcd3e7738c43f25c8f407ab63 . Therefore, the renaming isn't needed. Which sense does it make to make it public, if its not used publicly?